From f2bbeeef740d66f335a06e28a1cbb90f26568155 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Wed, 30 Nov 2022 15:24:55 +0530
Subject: [PATCH] Add mediatranscode sepolicy rules to public.

As mediatranscode update in google moving from private to public

Tracked-On: OAM-105081
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/caas/BoardConfig.mk b/caas/BoardConfig.mk
index 7d56d12..b0e250b 100644
--- a/caas/BoardConfig.mk
+++ b/caas/BoardConfig.mk
@@ -484,6 +484,7 @@ WITH_DEXPREOPT := true
 ##############################################################
 # Source: device/intel/mixins/groups/media/auto/BoardConfig.mk
 ##############################################################
+SYSTEM_EXT_PUBLIC_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/system_ext/public
 ##############################################################
 # Source: device/intel/mixins/groups/graphics/auto/BoardConfig.mk
 ##############################################################
-- 
2.38.1

